doc/SetupRSync.md

```diff
@@ -58,3 +58,8 @@ export RSYNC_PATH=/mnt/PIHDD/TeslaCam/
 Additional options for rsync over ssh can be configured using `~/.ssh/config` such as port number. To see all available options visit [the man page](https://linux.die.net/man/5/ssh_config).
 
 Stay in the `sudo -i` session return to the section "Set up the USB storage functionality" in the [main instructions](../README.md).
+
+# Step 3: Networking
+If you have a firewall between the server and teslausb you will need to allow ports 22 and ICMP.
+Port 22 is used for the rsync service
+ICMP is used to verify the archive is reachable from teslausb
```

Comment:
This doesn't appear to have anything to do with adding Signal notifications?
(the bit about icmp is also no longer correct)